osx/build.sh

```diff
@@ -177,12 +176,35 @@ test ! -d "$TEMP_OUTPUT_PATH" && mkdir "$TEMP_OUTPUT_PATH"
 ((STEP++))
 PYTHON_DOWNLOAD_URL="https://www.python.org/ftp/python/2.7.12/python-2.7.12-macosx10.6.pkg"
 cd $TEMP_OUTPUT_PATH
+echo "Downloading the minimum requirement of Python Installer at $PYTHON_DOWNLOAD_URL ..."
 wget --retry-connrefused --read-timeout=20 --waitretry=1 -t 100 --continue $PYTHON_DOWNLOAD_URL
 if [ $? -ne 0 ]; then
     echo ".. Abort!  Can't download Python at '$PYTHON_DOWNLOAD_URL'"
     exit 1
 fi
 
+KA_LITE_PROJECT_DIR="$SCRIPTPATH/KA-Lite"
+DMG_PATH="$OUTPUT_PATH/KA-Lite-Installer.dmg"
+DMG_BUILDER_PATH="$WORKING_DIR/create-dmg"
+CREATE_DMG="$DMG_BUILDER_PATH/create-dmg"
+KA_LITE_ICNS_PATH="$KA_LITE_PROJECT_DIR/KA-Lite/Resources/images/ka-lite.icns"
+
+test ! -d "$OUTPUT_PATH" && mkdir "$OUTPUT_PATH"
+
+((STEP++))
```

Comment:
Just a nitpick, you need to make an echo every time you increment the steps to give the user a feedback.  

With these changes, the user is notified only if the directory is non-existent.  If it exists already, the user won't know and will be shown a skipped step.

To simplify, just show an echo of the step description.